assets/javascripts/card-form-customization.js

```diff
@@ -58,6 +58,24 @@ function getDesignFormValues() {
   return formValues;
 }
 
+function handleFontChange() {
+  const fontName = document.getElementById('custom_sf_font_name');
+
+  if (fontName.value == 'Others') {
```

Comment:
1. Can we define the value representing other font names as a constant?
2. Shall we always use strict equality?

Reply:
We should. Done. Thanks.